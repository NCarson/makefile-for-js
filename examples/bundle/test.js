const foo = require('./subdir/Foo.js')
const find = require('find-node-modules')

console.log('hello from test.js')
console.log(foo)
console.log(find)
