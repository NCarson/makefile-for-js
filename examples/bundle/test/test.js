DIR = '../public/dist'

const bundle = require(DIR + '/bundle')
const vendor = require(DIR + '/vendor.min.js')


