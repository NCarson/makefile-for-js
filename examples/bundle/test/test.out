hello from test.js
1
[Function: findNodeModules]
